app/src/main/res/values/strings.xml

```diff
@@ -115,6 +116,32 @@
     <string name="shared_by_me_tab_name_">Shared by me</string>
     <string name="shared_with_me_tab_name">Shared with me</string>
 
+    <string name="public_archive_tab_name">"Public Archive"</string>
+    <string name="public_profile_tab_name">"Public Profile"</string>
+    <string name="public_profile_about">About</string>
+    <string name="public_profile_read_more">Read More</string>
+    <string name="public_profile_person_information">Person Information</string>
+    <string name="public_profile_full_name_hint">Full name</string>
+    <string name="public_profile_birth_date">Birth Date</string>
+    <string name="public_profile_birth_date_hint">YYYY-MM-DD</string>
+    <string name="public_profile_birth_location">Birth Location</string>
+    <string name="public_profile_birth_location_hint">Choose a location</string>
+    <string name="public_profile_online_presence">Online Presence</string>
+    <string name="public_profile_milestones">Milestones</string>
+    <string name="public_profile_about_text">Dr. Robert Friedman has over a decade of experience
```

Comment:
are these placeholders? this should not be included in the release

Reply:
yes, those are placeholders for Public Profile, I will take them out